master/internal/api_agents.go

```diff
@@ -121,3 +185,41 @@ func (a *apiServer) DisableSlot(
 		return resp, nil
 	}
 }
+
+func obstrufcateDevice(device *devicev1.Device) {
```

Comment:
nit: i'm not sure if we should leave these here on in the authz or agentrm package.

Reply:
I like the idea of moving it, that will allow use in other locations